src/FieldExchanger.jl

```diff
@@ -35,7 +35,7 @@ function import_atmos_fields!(csf, model_sims, boundary_space, turbulent_fluxes)
     end
 
     # surface density - needed for q_sat and requires atmos and sfc states, so it is calculated and saved in the coupler
-    Regridder.dummmy_remap!(csf.ρ_sfc, FluxCalculator.calculate_surface_air_density(atmos_sim, csf.T_S))
+    Regridder.dummmy_remap!(csf.ρ_sfc, FluxCalculator.calculate_surface_air_density(atmos_sim, csf.T_S)) # TODO: generalize for PartitionedStateFluxes (#445) (use individual T_S)
```

Comment:
I'd lean towards leaving TODOs like this out of the code and just keeping track of them via issues, especially before the release

Reply:
I take your point. For the time being, I would prefer keeping them in though, because they are quite useful for discussing our plans, while we discuss the interfaces this/next week. But just before the release we can/should go over and delete all of the TODOs in the repo and convert the remaining ones into issues.

Reply:
that sounds good to me!